proofs/sectorbuilder/rustsectorbuilder.go

```diff
@@ -77,8 +78,9 @@ type RustSectorBuilderConfig struct {
 	MetadataDir      string
 	MinerAddr        address.Address
 	SealedSectorDir  string
-	ProofsMode       proofs.Mode
+	SectorStoreType  proofs.Mode
```

Comment:
nit: why make this name change?  To me the previous name was easier to understand.

Reply:
Oops. That was a good catch. `ProofsMode` is no longer a field of the `RustSectorBuilderConfig` struct.